src/yio-model/mediaplayer/tvchannelmodel_mediaplayer.h

```diff
@@ -85,6 +85,7 @@ class ListTvChannelModel : public QAbstractListModel {
 class BrowseTvChannelModel : public QObject {
     Q_OBJECT
     Q_PROPERTY(QString id READ id NOTIFY idChanged)
+    Q_PROPERTY(QString time READ time NOTIFY idChanged)
```

Comment:
Invalid signal, this should be timeChanged

Reply:
Fixed